### PR TITLE
Bl or 691 mk

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient691.xml [hunk 1]
```diff
@@ -54,67 +239,101 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">243</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>365</height>
-                                        <width>235</width>
+                                    <dimensions type="outer" unit="in">
+                                        <height>14.0375</height>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient691.xml:243]: 0375 or 375?

File: LondonBritishLibrary/orient/BLorient691.xml [hunk 2]
```diff
@@ -31,21 +31,206 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                    <locus from="228r"/>
+                    <title type="complete" xml:lang="en" ref="LIT1895Martyr">Martyrdom of Eustathius, his wife and two children, under Marcianus (<date ref="ethiocal:Maskaram23">23rd of Maskaram</date>)</title>
+                    <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i20">
+                    <colophon xml:id="coloph1" xml:lang="gez">
+                    መልአ፡ መጽሐፈ፡ እንባእቆም፡ ነቢይ፡ ዐቢይ፡ አመ፡ ፲ወ፰፡ ለጥቅምት፡ በ<date calendar="grace" when="0477">፬፻፸፯: ዓመተ፡ ምሕረት፡</date> በመዋዕለ፡ <persName ref="PRS10215Yagbas">ያግባ፡<sic/> ጽዮን፡</persName> ንጉሥ፡ መፍቀሬ፡
+                    እግዚአብሔር፡ ዘተሠርዐ፡ በመዋዕሊሁ፡ ከመ፡ ይትአዘዙ፡ ሕዝበ፡ ኢትዮጵያ፡፡ ከመ፡ ይትዓገሱ፡ ከመ፡ ክርስቲያን፡ በከ፡ <gap reason="ellipsis" resp="PRS10171Wright"/>
+                    <note>The note is written by a later hand.</note>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient691.xml:223]: I wonder if this "colophon" should rather be reagarded as an addition, as it is written in a later hand and has little to do with a real colophon (the dating is rather a fantasy). What do you think @thea-m ?

File: LondonBritishLibrary/orient/BLorient691.xml [hunk 3]
```diff
@@ -54,67 +239,101 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                            
+<list>
+<item xml:id="a1">
+<locus target="#218v"/>
+<desc type="PurchaseNote">ለዝንቱ፡ መጽሐፍ፡ አጥረይክዎ፡ አነ፡ <persName ref="PRS13146SaggaKrestos">ጸጋ፡ ክርስቶስ፡</persName> በ፫ሸማ፡ ወ፬አሞሌ፡
+እምወለተ፡ አባ፡ <persName ref="PRS13147HablaSellase">ሐብለ፡ ሥላሴ፡</persName> ዘ<placeName ref="LOC1602Atrons">አትሮንሰ፡ ማርያም</placeName>፡፡
+መድኅንየሂ፡ <persName ref="PRS13172Ars">አርሳርዮስ፡</persName> ዘአትሮንሰ፡ ማርያም፡፡ ወመሳክርትሂ፡ በዥርወንድ፡
+<persName ref="PRS13165KeflaGiyorgis">ክፍለ፡ ጊዮርጊስ፤</persName> <persName ref="PRS13166TasfaIyasus">ተስፋ፡ ኢየሱስ፡</persName>
+ጸሓፈ፡ <persName ref="PRS13167Baselyos">ባስልዮስ፡</persName> አውቂያየ፡ ወዓሌ፡ አዛዥ፡ <persName ref="PRS13168Demyanos">ድምያኖስ፡</persName>
+ <persName ref="PRS13169YamanaKrestos">የማነ፡ ክርስቶስ፡</persName> አቤቶ፡ <persName ref="PRS13170EdaKrestos">እደ፡ ክርስቶስ፡</persName> ዘዋረብ፡፡</desc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient691.xml:277]: the role could be added to the personal names where possible

**Review by DariaElagina — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient691.xml (hunk 2 above)
Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient691.xml:223]: Yes, if it is added by a later hand I would also encode it as an addition.

File: LondonBritishLibrary/orient/BLorient691.xml [hunk 4]
```diff
@@ -31,21 +31,206 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                    <locus from="91v"/>
+                    <title type="complete" xml:lang="en" ref="LIT1651Homily">Homily of <persName ref="PRS5720JohnChr">John Chrysostom</persName>
+                    on the four beasts (<date ref="ethiocal:Hedar8">8th of Ḫǝdār</date>)</title>
+                    <incipit>ድርሳን፡ ዘብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ ሊቀ፡ ጳጳሳት፡ ዘቈስጠንጢንያ፡ በእንተ፡ ፬፡ እንስሳ፡ ዕበይት፡ መንፈሳዊያን፡ አመ፡ ሰሙኑ፡ ለኅዳር፡ ዘይትነበብ፡ በሰላመ፡ እግዚአብሔር፡ አሜን።</incipit>
+                    <textLang mainLang="gez"/>
+                    </msItem>
+
+                    <msItem xml:id="ms_i8">
+                    <locus from="106r"/>
+                    <title type="complete" xml:lang="en" ref="LIT6174Severus318">Treatise of <persName ref="PRS8788Severos">Severus (of Ashmūnain)</persName> on the Council of the 318 Fathers (at Nicaea)</title>
```
[the remark this answers is not in the document]
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient691.xml:108]: I give  the transliteration according to the EAe

File: LondonBritishLibrary/orient/BLorient691.xml (hunk 3 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient691.xml:332]: Other than owner, the role of these people is not clear to me.

File: LondonBritishLibrary/orient/BLorient691.xml [hunk 5]
```diff
@@ -54,67 +294,111 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+  <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+  <placeName ref="INS0101MadhaneAlam"/>.</desc>
+  <q xml:lang="gez">ገድለ፡ ጊዮርጊስ፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+  </item>
+
+  <item xml:id="a2">
+  <locus target="#214v"/>
+  <desc type="Unclear"> መልአ፡ መጽሐፈ፡ እንባእቆም፡ ነቢይ፡ ዐቢይ፡ አመ፡ ፲ወ፰፡ ለጥቅምት፡ በ<date calendar="grace" when="0477">፬፻፸፯: ዓመተ፡ ምሕረት፡</date> በመዋዕለ፡ <persName ref="PRS10215Yagbas">ያግባ፡<sic/> ጽዮን፡</persName> ንጉሥ፡ መፍቀሬ፡
+  እግዚአብሔር፡ ዘተሠርዐ፡ በመዋዕሊሁ፡ ከመ፡ ይትአዘዙ፡ ሕዝበ፡ ኢትዮጵያ፡፡ ከመ፡ ይትዓገሱ፡ ከመ፡ ክርስቲያን፡ በከ...>
+  </desc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient691.xml:337]: The incipit should be encoded in q with the seg. In the desc you can maybe summarize that this is a note on completion of the book in a later hand with a unrealistic dating

File: LondonBritishLibrary/orient/BLorient691.xml [hunk 6]
```diff
@@ -134,17 +418,22 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <term key="Hagiography"/>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient691.xml:418]: ChristianLiterature

File: LondonBritishLibrary/orient/BLorient691.xml (hunk 5 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient691.xml]: Sorry, I can not see that change.

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**